ts/Dashboards/Actions/Bindings.ts

```diff
@@ -183,6 +183,7 @@ namespace Bindings {
     /** @internal */
     export function componentFromJSON(
         json: Component.JSON,
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Better to remove the unused parameter?

Reply:
this one is marked with @todo so i assumed it was to be used eventually, but I can remove it 👍